api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -21,6 +21,7 @@
 import org.pmiops.workbench.db.dao.UserDao;
 import org.pmiops.workbench.db.dao.UserRecentResourceService;
 import org.pmiops.workbench.db.dao.UserService;
+import org.pmiops.workbench.firecloud.FireCloudServiceImpl;
```

Comment:
nit: Sorry for another nit on this, but should probably be on the service interface instead. The controllers shouldn't have a direct reference to the implementation classes.

Reply:
that's my bad. You gave me the same feedback before